sdk/servicebus/Microsoft.Azure.ServiceBus/CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 # Release History
+## 5.0.0
+### Breaking Changes
+- Enable a way to Unregister Message Handler and Session Handler [PR 14021](https://github.com/Azure/azure-sdk-for-net/pull/14021)
```

Comment:
This seems like it would be an additive change. Is there a reason it is considered breaking?

Reply:
It shouldn't be marked as breaking change. Instead it should go under "improvements"

---
In reply to: [486693049](https://github.com/Azure/azure-sdk-for-net/pull/15073#discussion_r486693049) [](ancestors = 486693049)